src/include/platform/CHIPDeviceConfig.h

```diff
@@ -592,6 +592,32 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT 0
 #endif
 
+/**
+ * CHIP_DEVICE_CONFIG_CRMP_RETRY_INTERVAL_IDLE
+ *
+ * Initial retransmission interval, or time to wait before retransmission after first
+ * failure in milliseconds.
+ *
+ * This is the default value, that might be adjusted by end device depending on its
+ * needs (e.g. sleeping period) using Service Discovery TXT record CRI key.
+ */
+#ifndef CHIP_DEVICE_CONFIG_CRMP_RETRY_INTERVAL_IDLE
```

Comment:
Note that we already have similar constants for that: `CHIP_CONFIG_RMP_DEFAULT_ACTIVE_RETRANS_TIMEOUT_TICK` and `CHIP_CONFIG_RMP_DEFAULT_INITIAL_RETRANS_TIMEOUT_TICK`. Maybe you could update comments and values of those instead of creating new ones?

Reply:
You are right, but honestly at this moment I can't figure out how to include headers from `message` module to the `lib/mdns`. The compilation structure and GN dependencies are so tangled that I don't have better idea than adding those intervals to OperationalAdvertisingParameters structure and pass it as an argument to the Advertise() method.